File: CsharpProjects/ObserverSolucao/ObserverSolucao/Subjects/Email.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using ObserverSolucao.Observers;

namespace ObserverSolucao.Subjects
{
    public class Email
    {
        public static void enviarEmail(Observer observer, string mensagem)
        {
            Console.WriteLine("-------------------------------------------------------------------------------------");
            Console.WriteLine("Email enviado para: " + observer.getNome() + " - " + observer.getEmail());
            Console.WriteLine("Mensagem :" + mensagem);
        }
    }
}

File: JavaProjects/ObserverSolucao/src/subjects/Email.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package subjects;

import observers.Observer;

/**
 *
 * @author jeffe
 */
public class Email {
    
    public static void enviarEmail(Observer observer, String mensagem){
        System.out.println("-------------------------------------------------------------------------------------");
        System.out.println("Email enviado para: " + observer.getNome() + " - " + observer.getEmail());
        System.out.println("Mensagem :" + mensagem);
    }
    
}
